public/client-v2/src/components/Feed.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from 'react';
+/* eslint-disable import/no-duplicates */
+import CAPISearchQuery from './CAPI/CAPISearchQuery';
+import { type Element } from './CAPI/CAPISearchQuery';
+/* eslint-enable import/no-duplicates */
+import ScrollContainer from './ScrollContainer';
+import FeedItem from './FeedItem';
+import Loader from './Loader';
+
+// TODO: get apiKey from context (or speak directly to FrontsAPI)
+
+const getThumbnail = (_elements: Element[]) => {
```

Comment:
All the logic that extract data from capi requests will probably lived in some shared file in the end as they will be used across the app but I guess we can start moving them when multiple components start making capi requests.

Reply:
I did do this but for some reason flow just started marking my types as `any` and I couldn't be bothered to figure it out 😞